doc/manuals/installation_and_administration_manual/gui.md

```diff
@@ -0,0 +1,431 @@
+#<a name="top"></a>Cosmos GUI
+
+Content:<br>
+
+* [Introduction](#section1)
+* [Installation](#section2)
+    * [Prerequisites](#section2.1)
+    * [Installing the GUI](#section2.2)
+    * [Installing the database](#section2.3)
+        * [Upgrading from 0.1.0 to 0.2.0](#section2.3.1)
+        * [Upgrading from 0.2.0 to 0.3.0](#section2.3.2)
+    * [Registering the application in the Identity Manager](#section2.4)
+    * [Unit tests](#section2.5)
+* [Configuration](#section3)
+* [Running](#section4)
+* [Administration](#section5)
+    * [Logging traces](#section5.1)
+    * [Database](#section5.2)
+* [Annexes](#section6)
+    * [Annex A: Creating and installing a RSA identity](#section6.1)
+    * [Annex B: Creating a self-signed certificate](#section6.2)
+    * [Annex C: Binding the GUI to a port under TCP/1024](#section6.3)
+
+##<a name="section1"></a>Introduction
+
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/214/commits/a37133d55f651e5407b3eb87fb9192bc1a85c89e